mart/conf/SiteDefs.pm

```diff
@@ -27,6 +27,8 @@ sub update_conf {
   $SiteDefs::ENSEMBL_MART_PLUGIN_ENABLED    = 1;
   $SiteDefs::PERL_RLIMIT_AS                 = '8192:16384';
 
+  $SiteDefs::ENSEMBL_TMP_MESSAGE_FILE		    = defer { $SiteDefs::ENSEMBL_TMP_ROOT.'/ensembl_tmp_message' }; # Temporary message
```

Comment:
This change is not needed. It will use the one in ensembl-webcode

Reply:
Did we not decide to put these here as mart doesn't always use ensembl-webcode?

Reply:
V mart always use ensembl-webcode. It's NV marts that don't use it, NV marts don't use public-plugins also.

Reply:
Okay, cool. I'll remove it.